idseq/__init__.py

```diff
@@ -9,45 +13,54 @@ def validate_file(path, name):
     pattern = uploader.INPUT_REGEX
     if not re.search(pattern, path):
         print(
-            "ERROR: %s (%s) file does not appear to be a fastq or fasta file." %
-            (name, path))
-        print("ERROR: Basename expected to match regex '%s'" % pattern)
+            "ERROR: %s (%s) file does not appear to be a fastq or fasta file."
+            % (name, path))
+        print(
+            "Accepted formats: fastq/fq, fasta/fa, fastq.gz/fq.gz, fasta.gz/fa.gz"
+        )
         raise ValueError
 
 
 def main():
     parser = argparse.ArgumentParser(
-        description='Submit a sample to idseq. (Accepts fastq or fasta files, single or paired, gzipped or not.)')
+        description='Submit a sample to idseq. (Accepts fastq or fasta files, single or paired, gzipped or not.)'
+    )
 
     parser.add_argument(
         '-p',
         '--project',
         metavar='name',
         type=str,
-        required=True,
         help='Project name. Make sure the project is created on the website ')
     parser.add_argument(
         '-s',
         '--sample-name',
         metavar='name',
         type=str,
         help='Sample name. It should be unique within a project')
-    parser.add_argument('-u', '--url', metavar='url', type=str, required=True,
-                        help='idseq website url: i.e. dev.idseq.net')
+    parser.add_argument(
+        '-u',
+        '--url',
+        metavar='url',
+        type=str,
+        help='idseq website url: i.e. dev.idseq.net')
```

Comment:
should probably default to prod site? We are the only ones who might want to use staging instead